static/src/js/containers/ShapefileDropzoneContainer/__tests__/ShapefileDropzoneContainer.test.js

```diff
@@ -210,7 +226,239 @@ describe('ShapefileDropzoneContainer component', () => {
+        expect(props.onShapefileErrored).toHaveBeenCalledWith({
+          message: 'To use a shapefile, please upload a zip file that includes its .shp, .shx, and .dbf files.'
+        })
+      })
+
+      test('fires the correct callbacks for .shx files', () => {
+        const { enzymeWrapper, props } = setup()
+
+        enzymeWrapper.find('WithDropzone').props().onError({
+          name: 'test-file.shx'
+        })
+
+        expect(props.onToggleShapefileUploadModal).toHaveBeenCalledTimes(1)
+        expect(props.onToggleShapefileUploadModal).toHaveBeenCalledWith(false)
+
+        expect(props.onShapefileErrored).toHaveBeenCalledTimes(1)
+        expect(props.onShapefileErrored).toHaveBeenCalledWith({
+          message: 'To use a shapefile, please upload a zip file that includes its .shp, .shx, and .dbf files.'
+        })
+      })
+
+      test('fires the other filtype when zip/shp/dbf/shx is in the filename but not the filepath', () => {
+        const { enzymeWrapper, props } = setup()
+
+        enzymeWrapper.find('WithDropzone').props().onError({
+          name: 'zip-shp-dbf-shx.java'
+        })
+
+        expect(props.onToggleShapefileUploadModal).toHaveBeenCalledTimes(1)
+        expect(props.onToggleShapefileUploadModal).toHaveBeenCalledWith(false)
+
+        expect(props.onShapefileErrored).toHaveBeenCalledTimes(1)
+        expect(props.onShapefileErrored).toHaveBeenCalledWith({
+          message: 'Invalid file format.'
+        })
+      })
+    })
+
+    describe('when given a Keyhole Markup Language file', () => {
+      test('fires the correct callbacks for .kml files', () => {
+        const { enzymeWrapper, props } = setup()
+
+        enzymeWrapper.find('WithDropzone').props().onError({
+          name: 'test-file.kml'
+        })
+
+        expect(props.onToggleShapefileUploadModal).toHaveBeenCalledTimes(1)
+        expect(props.onToggleShapefileUploadModal).toHaveBeenCalledWith(false)
+
+        expect(props.onShapefileErrored).toHaveBeenCalledTimes(1)
+        expect(props.onShapefileErrored).toHaveBeenCalledWith({
+          message: 'To use a Keyhole Markup Language file, please upload a valid .kml or .kmz file.'
+        })
+      })
+
+      test('fires the correct callbacks for .kmz file', () => {
+        const { enzymeWrapper, props } = setup()
+
+        enzymeWrapper.find('WithDropzone').props().onError({
+          name: 'test-file.kmz'
+        })
+
+        expect(props.onToggleShapefileUploadModal).toHaveBeenCalledTimes(1)
+        expect(props.onToggleShapefileUploadModal).toHaveBeenCalledWith(false)
+
+        expect(props.onShapefileErrored).toHaveBeenCalledTimes(1)
+        expect(props.onShapefileErrored).toHaveBeenCalledWith({
+          message: 'To use a Keyhole Markup Language file, please upload a valid .kml or .kmz file.'
+        })
+      })
+
+      test('fires the other filtype when kml/kmz is in the filename but not the filepath', () => {
```

Comment:
files the other filtype?

Its a typo in all the new tests you added

Reply:
What i mean by this is that it's fires the error that's used for "other" filetypes, meaning, filetypes that aren't included in the list of acceptable filetypes.